src/utils/text.js

```diff
@@ -39,4 +39,24 @@ function formatKeyShortcut(shortcut: string): string {
     .replace(/Shift\+/g, "Shift ");
 }
 
-export { formatKeyShortcut };
+/**
+ * Truncates the received text to the maxLength in the format:
+ * Original: 'this is a very long text and ends here'
+ * Truncated: 'this is a ver...nd ends here'
+ * @param {String} text - Original text
+ * @param {Number} maxLength - Max allowed length
+ * @memberof utils/text
+ * @static
+ */
+function truncateText(text: string, maxLength: number): string {
+  let truncatedText = text;
+  if (text.length > maxLength) {
+    truncatedText = `${text.substring(
+      0,
+      Math.round(maxLength / 2) - 2
+    )}...${text.substring(text.length - Math.round(maxLength / 2 - 2))}`;
+  }
+  return truncatedText;
+}
+
+export { formatKeyShortcut, truncateText };
```

Comment:
looks like we have endTruncateStr
https://github.com/devtools-html/debugger.html/blob/master/src/utils/utils.js#L41-L46

should we move this there and rename both to truncateEnd, truncateMiddle?

Reply:
good call